### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by ufukcalimli**

### CODE REVIEW

**Comment by ufukcalimli** [comment]
Now I removed the commit from week1, but I have got to solve some small issues on github. That's why last two commits have same codes.

File: Week2/homework/maartjes-work.js [hunk 1]
```diff
@@ -47,15 +47,16 @@ const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const hours = tasks.map(task => task.duration / 60);
+  return hours.filter(hour => hour >= 2).reduce((sum, hour) => sum + hour * hourlyRate, 0);
 }
```
Comment (remarcmij) [inline Week2/homework/maartjes-work.js]: With the parameter names you chose for the array methods you clearly demonstrate your understand what happens to the data as it get processed while traversing the method chain. Good show! But you can combine this all in one big chain like this:

```js
function computeEarnings(tasks, hourlyRate) {
  return tasks
    .map(task => task.duration / 60)
    .filter(hour => hour >= 2)
    .reduce((sum, hour) => sum + hour * hourlyRate, 0);
}
```

File: Week2/homework/maartjes-work.js [hunk 2]
```diff
@@ -47,15 +47,16 @@ const maartjesHourlyRate = 20;
...
 
 // eslint-disable-next-line no-unused-vars
 const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
 
 // add code to convert `earnings` to a string rounded to two decimals (euro cents)
 
-console.log(`Maartje has earned €${'replace this string with the earnings rounded to euro cents'}`);
+console.log(`Maartje has earned €${earnings.toLocaleString()}`);
```
Comment (remarcmij) [inline Week2/homework/maartjes-work.js]: The number on my computer is printed with 3 decimals. This does not meet the requirement of the assignment.

File: Week2/homework/maartjes-work.js [hunk 3]
```diff
@@ -47,16 +47,20 @@ const maartjesHourlyRate = 20;
...
 }
 
 // eslint-disable-next-line no-unused-vars
 const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
 
 // add code to convert `earnings` to a string rounded to two decimals (euro cents)
 
-console.log(`Maartje has earned €${earnings.toLocaleString()}`);
+console.log(
+  `Maartje has earned €${earnings.toLocaleString(undefined, { maximumFractionDigits: 2 })}`
```
Comment (remarcmij) [inline Week2/homework/maartjes-work.js]: Simpler still is `earnings.toFixed(2)`.

**Pull request closed without merging**